sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -444,10 +449,48 @@ protected void defaultConfigure() {}
         final AbstractConfigurationModule cliOverrideModule = new AbstractConfigurationModule() {
             @Override
             protected void defaultConfigure() {
-                cmdLine.getDeviceEpr().ifPresent(epr -> {
+                final String epr = cmdLine.getDeviceEpr();
+                if (epr != null) {
                     LOG.info("Using consumer target epr from cli: {}", epr);
```

Comment:
In the config.toml, the setting is named "DeviceEpr". Given that SDCcc is testing Providers only, I would consider calling it "consumer target epr" as rather confusing. The same holds for the other settings below ("consumer location facility", "consumer location building", "consumer location point of care", "consumer location floor" and "consumer location room" are even missing the word "target", which makes it even more confusing)

Reply:
changed it